db/dev_seeds/sdg.rb

```diff
@@ -1,3 +1,35 @@
 section "Creating Sustainable Development Goals" do
   load Rails.root.join("db", "sdg.rb")
+
+  SDG::Target.sample(30).each do |target|
+    title = "Title for default locale"
+    description = "Description for default locale"
+    rand(2..3).times do |n|
+      local_target = SDG::LocalTarget.create!(code: "#{target.code}.#{n}",
+                                              title: title,
+                                              description: description,
+                                              target: target)
+      random_locales.map do |locale|
+        Globalize.with_locale(locale) do
+          local_target.title = "Title for locale #{locale}"
+          local_target.description = "Description for locale #{locale}"
+          local_target.save!
+        end
+      end
+    end
+  end
+
+  relatables = [Debate.last, Proposal.last, Poll.last, Legislation::Process.last, Budget::Investment.last]
+  relatables.each do |relatable|
+    3.times do
```

Comment:
The test failed because the `sample` method returned the same thing twice :thinking:.

How about:

```ruby
    SDG::Goal.sample(3).each do |goal|
      target = goal.targets.sample
      # ...

Reply:
Yes, I did not realize this until the test crashed.